.github/workflows/continuous-deployment.yml

```diff
@@ -177,61 +192,6 @@ jobs:
             |<span aria-hidden="true">🔍</span> Latest deploy log | [${{ steps.job-url.outputs.html_url }}](${{ steps.job-url.outputs.html_url }}) |
           comment_tag: feature-branch-deployment-status
 
-      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
```

Comment:
you need this checkout that the deployment uses the helm chart of the branch to deploy

Reply:
Opps, missed that done, thanks. Added back in https://github.com/ecamp/ecamp3/pull/3363/commits/cf09b190fa9e38cdb91b0acd7a6cd058d69cb7bc